directory.build.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+  <ItemGroup>
+    <PackageReference
+            Include="Microsoft.NETFramework.ReferenceAssemblies"
+            IncludeAssets="analyzers;build;buildMultitargeting;buildTransitive"
+            PrivateAssets="All"
+        />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" IncludeAssets="analyzers;build;buildMultitargeting;buildTransitive" PrivateAssets="All" />
```

Comment:
SourceLink can be under Directory.build.props file, doesn't have to be under .targets file

Reply:
copy paste error.